### ISSUE TITLE
Player play/pause issues on the remote

### ISSUE THREAD

**Issue by nklerk** [NEEOInc/neeo-sdk#119]
While pressing the play/pause button(switch) using the player ui element, the state switches back to its original state. On EUI it works as expected where based on the player state the correct button is shown.
In my code i also change the description of the playing content to PAUSED while the content is being paused. this doesn't seem to work on TR2 as well.

**Code Used:**
https://github.com/nklerk/neeo_driver-kodi/tree/player

**Driver Used:**
Driver Kodi (Player)

**EUI Working normal example:**
https://youtu.be/5-z5Wy-lh6o

**TR2 weird behavior example:**
https://youtu.be/-gcFn0gOYV4

**Comment by MichaelKohler** [comment]
@nklerk, thanks for filing this issue. Do you see any logs from the button press when pressing it on the TR2? Does Kodi stop? As far as I can see right now (on 0.51.x), the pause does not work at all. Therefore it also won't update the gui data state and therefore fall back to the initial state.

Happy to hear if you agree with my initial analysis or whether I'm missing something or not. I'm gonna test this on our current development firmware and will file an internal issue for this if necessary.

**Comment by nklerk** [comment]
I’m unable to do tests this week. I’ll update the coming weekend.

**Comment by MichaelKohler** [comment]
@nklerk I've also tested this with our example. Can you verify that this problem doesn't exist with the example once you have time? Thanks!

**Comment by nklerk** [comment]
I've added the "PLAY PAUSE" button. Here are the updated kodi tests.

Here is a Kodi demo of the TR2 not having a working PLAY/PAUSE. https://youtu.be/uv3cQXBwW5M The OK button once the "PLAY PAUSE" button was added does it's job. also visible in the video.

On EUI the Play/Pause button just works normal.
https://youtu.be/2Tyt5p_FT08

on the player example the TR2 play/pause toggle seem to do something but to say that it works is maybe a great word. lol.. the OK button does nothing though.
https://youtu.be/gtuAXRDhSdw

I'll leave it for what it is as i've chosen not to use the player and as the Example seem to behave different than my player i'll close the issue.
